src/checkbox/__tests__/checkbox.test.tsx

```diff
@@ -3,8 +3,10 @@
 import React, { useState } from 'react';
 import { render } from '@testing-library/react';
 import createWrapper, { CheckboxWrapper } from '../../../lib/components/test-utils/dom';
+import FormField from '../../../lib/components/form-field';
 import Checkbox, { CheckboxProps } from '../../../lib/components/checkbox';
 import styles from '../../../lib/components/internal/components/checkbox-icon/styles.selectors.js';
+import abstractSwitchStyles from '../../../lib/components/internal/components//abstract-switch/styles.css.js';
```

Comment:
Double slash in this import. Even if this works, better to keep it clean

```suggestion
import abstractSwitchStyles from '../../../lib/components/internal/components/abstract-switch/styles.css.js';
```